lib/src/facts/linux/networking_resolver.cc

```diff
@@ -71,4 +89,85 @@ namespace facter { namespace facts { namespace linux {
         return interface;
     }
 
+    void networking_resolver::read_routing_table()
+    {
+        auto ip_command = lth_exe::which("ip");
+        if (ip_command.empty()) {
+            LOG_DEBUG("Could not find the 'ip' command. Network bindings will not be populated from routing table");
+            return;
+        }
+
+        auto parse_route_line = [](string& line, std::vector<route>& routes) {
+            vector<boost::iterator_range<string::iterator>> parts;
+            boost::split(parts, line, boost::is_space(), boost::token_compress_on);
+            if (parts.size() % 2 != 1) {
+                LOG_WARNING("Could not process routing table entry: Expected a destination followed by key/value pairs, got '{}'", line);
```

Comment:
This is mostly a guard against a future version of the `ip` command changing its output format, so obviously never actually came up in my testing